src/extension.ts

```diff
@@ -5,13 +5,14 @@ import * as server from './server';
 import { RSPClient } from 'rsp-client';
 import { ExtensionAPI, CommandHandler } from './extensionApi';
 
-const client = new RSPClient('localhost', 27511);
+let client;
```

Comment:
This might never get initialized in case rsp-server could not start and threw an error. Then in deactivate method the line
``` 
client.shutdownServer();
```

would fail.